cmd/minishift/cmd/delete.go

```diff
@@ -36,6 +37,9 @@ var deleteCmd = &cobra.Command{
 }
 
 func runDelete(cmd *cobra.Command, args []string) {
+	instancedata.Instance().ReadInstanceData()
```

Comment:
> If not as part of the config directory, we could also use the machines directory (not the actual libmachine managed directory minshift). We could call it minishift.json or more generic <machine-name>.json

Same trade off here, we still need to handle cleanup part. 

This place is also fine since we know what to cleanup and easier to do then config place IMO.